tests/twitter.test.ts

```diff
@@ -8,41 +8,107 @@ afterEach(() => server.resetHandlers());
 afterAll(() => server.close());
 
 describe("Twitter", () => {
-  test("Throw on missing env variable", async () => {
-    // Save the env first to test throws with missing env values
-    const savedEnv = process.env;
+  test("Throw on missing all env variables", async () => {
+    // Save the current environment variables
+    const originalEnv = process.env;
+
+    process.env = {
+      ...originalEnv,
+      TWITTER_API_KEY: undefined,
+      TWITTER_API_KEY_SECRET: undefined,
+      TWITTER_ACCESS_TOKEN: undefined,
+      TWITTER_ACCESS_TOKEN_SECRET: undefined,
+    };
+
+    // Use jest.resetModules to ensure the module is reloaded with new env vars
+    jest.resetModules();
+
     await expect(async () => {
-      dotenv.config({
-        override: true,
-      });
-      process.env.TWITTER_API_KEY = "";
-      await import("../helpers/twitter");
-    }).rejects.toThrow();
+      const { default: twitterHelper } = await import("../helpers/twitter");
+      console.log(twitterHelper);
```

Comment:
```suggestion
```

Reply:
I did this for two reasons the first was occasionally it was failing and it was because it was being imported but the not doing anything with it was likely the cause of it failing and the second was to silence the unused var error.

I tried it again without doing anything with the import and it passed each time but I've replaced it with `() => twitterHelper`. 1. to stop the unused error and 2, just in case doing nothing with the import causes nothing to happen in the test